packages/v4/patternfly-docs/content/accessibility/menu-toggle/menu-toggle.md

```diff
@@ -0,0 +1,56 @@
+
+
+## Testing
+
+At a minimum, a menu toggle component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-1" label={<span>Users should be able to use keyboard navigation to activate the menu toggle with <kbd>Enter</kbd> or <kbd>Space</kbd> to enter a menu.</span>} description={<span><kbd>Tab</kbd> navigates to the menu and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element. Refer to our <a href="/components/menu/accessibility">menu accessibility documentation</a> for more implementation details when using these components together. </span>}/>
+  </ListItem>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-2" label={<span>If there is not text within the menu toggle, such as a kebab menu toggle, an <code className="ws-code">aria-label</code> is used on the <code className="ws-code">MenuToggle</code>.</span>} description="This will communicate what the menu toggle is to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-3" label={<span>Any action used within a menu toggle has an <code className="ws-code">aria-label</code>.</span>} description="This will differentiate each action when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+</List>
+
+
+## React customization
+
+The following React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label="[text describing the menu toggle]"` | `MenuToggle` | Adds an accessible name to the menu toggle. |
+| `icon={[custom icon]}` | `MenuToggle` | Adds an optional icon rendered inside the toggle, before the children content. Add `aria-hidden` to the icon to hide the decorative item from screen reader elements. |
+| `aria-label="[text describing the action]"` | `MenuToggleAction` | Adds an accessible name to the menu toggle action. |
+| `aria-label="[text labeling the checkbox]"` | `MenuToggleCheckbox` | Adds an accessible name to the checkbox item on the menu toggle. |
+
+
+
+## HTML/CSS customization
+
+The following HTML attributes and PatternFly classes can be used for more fine-tuned control over accessibility.
+
+| Attribute or Class | Applied to | Reason |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-menu-toggle`, `.pf-c-menu-toggle__button` | Indicates that the menu toggle component is in the expanded state. |
+| `aria-expanded="false"` | `.pf-c-menu-toggle`, `.pf-c-menu-toggle__button` | Indicates that the menu toggle component is in the collapsed state. |
+| `aria-label="[text describing the menu toggle]"` | `.pf-c-menu-toggle.pf-m-plain` | Gives the plain menu toggle component an accessible label. |
```

Comment:
I don't think the aria-label here should only be used when the `pf-m-plain` modifier is used. It'd be more accurate to say "An accessible label for the menu toggle when there is no visible text within the toggle. Required when using a `pf-m-plain` with no text.

Reply:
Agreed, another one that I copied from core